### ISSUE TITLE
Thunderstorm while sapped.

### ISSUE THREAD

**Issue by kysyourself** [dalaranwow/dalaran-wow#5009]
[/thunderstorm/]: # (INFORMATION ABOUT TAGS)
[/thunderstorm/]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[/thunderstorm/]: # (Priority - ... tags are assigned by team members)
[/thunderstorm/]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[/thunderstorm/]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[/thunderstorm/]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[/thunderstorm/]: # (***************************************************************)
[/thunderstorm/]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[/thunderstorm/]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. thunderstorm
2. thunderstorm
3. thunderstorm

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by robinsch** [comment]
> Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
